galera/src/ist.cpp

```diff
@@ -204,9 +216,102 @@ IST_determine_recv_addr (gu::Config& conf)
         recv_addr += ":" + gu::to_string(port);
     }
 
+    log_info << "IST receiver addr using " << recv_addr;
     return recv_addr;
 }
 
+static std::string IST_determine_recv_bind(gu::Config& conf) {
+	std::string recv_bind;
+
+	try
+	{
+		recv_bind = conf.get(galera::ist::Receiver::RECV_BIND);
+	}
+	catch (gu::NotFound&)
```

Comment:
I think that since you add this parameter to config set above, here you must be catching gu::NotFound is pointless.
Also, instead of duplicating the whole process of getting recv_addr from config, you could just pass already determined recv_addr to this function, or see below